src/utils/CourseChecklist/courseChecklistValidators.js

```diff
@@ -64,7 +64,7 @@ export const hasDiverseSequences = (subsections) => {
 };
 
 export const hasWeeklyHighlights = sections => (
-  sections.highlights_enabled
+  sections.highlights_active_for_course
```

Comment:
For the logic here, we will want to do `sections.highlights_active_for_course && sections.highlights_enabled` as this validation is checking to see if the course team has turned on weekly highlights in their course. So the check will first validate that highlights are active, then if the course team is using them.